core/src/main/java/com/datastrato/gravitino/meta/ManagedUser.java

```diff
@@ -0,0 +1,460 @@
+/*
+ * Copyright 2024 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+package com.datastrato.gravitino.meta;
+
+import com.datastrato.gravitino.Auditable;
+import com.datastrato.gravitino.Entity;
+import com.datastrato.gravitino.Field;
+import com.datastrato.gravitino.HasIdentifier;
+import com.datastrato.gravitino.NameIdentifier;
+import com.datastrato.gravitino.authorization.User;
+import com.google.common.collect.Maps;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import lombok.ToString;
+
+/** A class representing a user metadata entity in Gravitino. */
+@ToString
+public class ManagedUser implements User, Entity, Auditable, HasIdentifier {
+
+  public static final Field ID =
+      Field.required("id", Long.class, " The unique id of the user entity.");
+  public static final Field NAME =
+      Field.required("name", String.class, "The name of the user entity.");
+  public static final Field PROPERTIES =
+      Field.optional("properties", Map.class, "The properties of the user entity.");
+  public static final Field AUDIT_INFO =
+      Field.required("audit_info", AuditInfo.class, "The audit details of the user entity.");
+
+  public static final Field COMMENT =
+      Field.optional("comment", String.class, "The comment of the user entity.");
+
+  public static final Field FIRST_NAME =
+      Field.required("first_name", String.class, "The first name of the user entity.");
+
+  public static final Field LAST_NAME =
+      Field.required("last_name", String.class, "The last name of the user entity");
+
+  public static final Field DISPLAY_NAME =
+      Field.required("display_name", String.class, "The display name of the user entity");
+
+  public static final Field EMAIL_ADDRESS =
+      Field.required("email_address", String.class, "The email address of the user entity");
+
+  public static final Field ACTIVE =
+      Field.required(
+          "active", Boolean.class, "The status of the user entity is whether active or not.");
+
+  public static final Field GROUPS =
+      Field.optional("groups", List.class, "The groups of the user entity");
+
+  public static final Field ROLES =
+      Field.optional("roles", List.class, "The roles of the user entity");
+
+  public static final Field DEFAULT_ROLE =
+      Field.optional("default_role", String.class, "The default role of the user entity");
```

Comment:
Can you please either group them altogether like above, or add a blank at each field?